<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TD Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,1">
      <item>
       <widget class="MapView" name="mapView_">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4">
        <item>
         <widget class="SelectedView" name="selectedView_"/>
        </item>
        <item>
         <widget class="TilesView" name="tilesView_"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_"/>
    <addaction name="actionOpen_"/>
    <addaction name="actionSave_"/>
    <addaction name="actionSaveAs_"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMap_">
    <property name="title">
     <string>&amp;Map</string>
    </property>
    <addaction name="actionShowGrid_"/>
   </widget>
   <addaction name="menuFile_"/>
   <addaction name="menuMap_"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionNew_">
   <property name="text">
    <string>&amp;New...</string>
   </property>
  </action>
  <action name="actionOpen_">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionSave_">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSaveAs_">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionShowGrid_">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Grid</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TilesView</class>
   <extends>QGraphicsView</extends>
   <header>tilesview.h</header>
  </customwidget>
  <customwidget>
   <class>SelectedView</class>
   <extends>QGraphicsView</extends>
   <header>selectedview.h</header>
  </customwidget>
  <customwidget>
   <class>MapView</class>
   <extends>QGraphicsView</extends>
   <header>mapview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
